portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2771,6 +2778,92 @@ private long _getValidFileEntryTypeId(
 		}
 	}
 
+	private void _inheritRolesPermissions(
+			long propagationParentFolderId, DLFileEntry dlFileEntry)
```

Comment:
Should be `parentFolderId` or `inheritableParentFolderId`

Reply:
Fit it. Thanks